apps/fluent-tester/src/FluentTester/TestComponents/Common/MenuPicker.windows.tsx

```diff
@@ -0,0 +1 @@
+export * from './MenuPicker.desktop';
```

Comment:
I think this will fail because FURN windows doesn't have `Callout`, which `Menu` and therefore `MenuPicker` depend on. You'll probably have to delete this and make windows use the community picker